x-pack/plugins/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -51,6 +64,23 @@ export class OpenAIConnector extends SubActionConnector<Config, Secrets> {
     this.provider = this.config.apiProvider;
     this.key = this.secrets.apiKey;
 
+    this.openAI =
+      this.config.apiProvider === OpenAiProviderType.AzureAi
+        ? new OpenAI({
+            apiKey: this.secrets.apiKey,
+            baseURL: this.config.apiUrl,
+            defaultQuery: { 'api-version': getAzureApiVersionParameter(this.config.apiUrl) },
+            defaultHeaders: { 'api-key': this.secrets.apiKey },
+          })
+        : new OpenAI({
+            // default to empty string for unit tests x-pack/plugins/actions/server/integration_tests/connector_types.test.ts
+            apiKey: this.secrets.apiKey ?? '',
+            // without, this test fails x-pack/plugins/actions/server/integration_tests/connector_types.test.ts
+            // it seems the OpenAI sdk thinks the server emulation in jest is a browser environment
+            // without setting this flag, the sdk throws an error and the test will fail
+            dangerouslyAllowBrowser: true,
```

Comment:
Please also delete that property (and comments)!!! :-)

Reply:
done!